contracts/StakeManagerV2.sol

```diff
@@ -306,15 +298,73 @@ contract StakeManagerV2 is
         emit Delegated(preferredOperatorAddress, msg.value);
     }
 
+    /// @dev internal fn to check if new exchange rate is within limits
+    function _checkIfNewExchangeRateWithinLimits(uint256 currentER) internal {
+        uint256 maxAllowableER = maxExchangeRateSlippageBps * currentER / 10_000;
+        uint256 newER = convertBnbXToBnb(1 ether);
+        if (newER > maxAllowableER) {
```

Comment:
What about ER decrease due to slashing?

Reply:
for ER decrease, we can't do anything, that should be okay ?
we should show the correct ER.

Reply:
Okay what can we do in case of ER increase? also how to unpause it and update our ER?

Reply:
updated the logic as discussed, please review again